examples/lib/BUILD.bazel

```diff
@@ -6,6 +6,9 @@ npm_package(
         "index.js",
         "package.json",
     ],
-    package = "@mycorp/mylib",
+    # `package = "@mycorp/mylib"` does not need to be specified since the name
+    # that this package is linked as is set by the pnpm lockfile. If it is
```

Comment:
should/could it print a warning/error if you *do* specify package in a place where the value will be ignored?

Reply:
will make an issue to do as a follow-up